tests/JavaTest.java

```diff
@@ -129,6 +129,12 @@ static void TestBuffer(ByteBuffer bb) {
         TestEq(monster.test4Length(), 2);
         TestEq(test_0.a() + test_0.b() + test_1.a() + test_1.b(), 100);
 
+	Test.Vector test4Vector = monster.test4Vector();
+        test_0 = test4Vector.get(0);
```

Comment:
Aren't indexing functions in Java typically called `at`? Or is `get` more typical?

Reply:
`get` is used most often.